tests/lax_numpy_test.py

```diff
@@ -2648,6 +2648,28 @@ def testSort(self, dtype, shape, axis):
     self._CheckAgainstNumpy(jnp_fun, np_fun, args_maker)
     self._CompileAndCheck(jnp_fun, args_maker)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_{}_axis={}".format(
+          jtu.format_shape_dtype_string(shape, dtype), axis),
+        "shape": shape, "dtype": dtype, "axis": axis}
+      for dtype in all_dtypes
+      for shape in one_dim_array_shapes
+      for axis in [None]))
+  def testSortComplex(self, dtype, shape, axis):
+    # TODO(b/141131288): enable test once complex sort is supported on TPU.
+    if (jnp.issubdtype(dtype, jnp.complexfloating)
+        and jtu.device_under_test() == "tpu"):
+      self.skipTest("complex sort not supported on TPU")
+    rng = jtu.rand_some_equal(self.rng())
+    args_maker = lambda: [rng(shape, dtype)]
+    jnp_fun = jnp.sort_complex
+    np_fun = np.sort_complex
+    if axis is not None:
+      jnp_fun = partial(jnp_fun, axis=axis)
+      np_fun = partial(np_fun, axis=axis)
```

Comment:
`sort_complex` doesn't take an `axis` argument, so you can remove this logic (and remove all mentions of `axis` above).

Since there are no static arguments, you can remove the ``jnp_fun``/``np_fun`` lines and put the functions directly in the checks; i.e.
```python
self._CheckAgainstNumpy(jnp.sort_complex, np.sort_complex, args_maker)
self._CompileAndCheck(jnp.sort_complex, args_maker)
```